1397/en.md

```diff
@@ -0,0 +1,162 @@
+# Sudoku Solver
+
+###### Helpful Resources:
+
+[Sudoku Solver Geeks For Geeks](https://www.geeksforgeeks.org/sudoku-backtracking-7/)
+
+[Understanding Sudo solves in CPP](https://www.youtube.com/watch?v=lIOupNAafT0)
+
+[Introduction to Backtracking](https://www.youtube.com/watch?v=DKCbsiDBN6c)
+
+###### Solution Approach
+
+Firstly we implement a basic sudoku solver with four important points:
+
+1)We first have to find the empty positions for that we have the findEmptyLocation function
+
+2) To solve a sudoku we have to place a number at an empty position denoted by '.' in this question
+
+3) We use the isSafe function here to determine whether it is safe to place a particular number
+
+4) If empty location found, we iterate from 1 to 9 and try to place that number in that empty position, checking whether it is safe to place the value in each iteration
+
+5) If the empty positions are such that the number of values that needs to be placed exceeds 9 for the given set of isSafe conditions which means we can't just place 1 to 9, then we can conclude that the Sudoku is not solvable and the empty positions remain as 0 denoting that it cannot be filled
```

Comment:
> the number of values that needs to be placed exceeds 9 for the given set of isSafe conditions which means

Implementation detail. A bit confusing if someone doesn't look at the source code. Simply chop this portion off.